test/markets/markets.e2e-spec.ts

```diff
@@ -0,0 +1,66 @@
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MainModule } from '../../src/main.module';
+
+const request = require('supertest');
+
+describe('Markets', () => {
+
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [MainModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+
+    // Validation pipeline is require to check validations.
+    app.useGlobalPipes(
+      new ValidationPipe({
+        whitelist: true,
+        transform: true,
+        forbidNonWhitelisted: true,
+        transformOptions: {
+          enableImplicitConversion: true,
+        },
+      }),
+    );
+    await app.init();
+  });
+
+  it(`GET /markets`, () => {
+    return request(app.getHttpServer()).get('/markets')
+      .expect(200)
+      .expect('Content-Type', 'application/json; charset=utf-8')
+      .expect(require('./expected-responses/GET-markets.json'));
```

Comment:
1. can we import the file at the top and assign it to a variable? 
2. the `expect` part should be something like `expect(response)`.toEqual(get-markets-expectation);` - see this page https://jestjs.io/docs/expect
3. can we execute the `expect` statements on separate lines? i.e. 
`expect(response.body)`.toEqual(get-markets-expectation);`
`expect(response.status)`.toEqual({status: 200});`
or something like that ^

Reply:
1. Done
2. NestJs mostly uses chained checks in samples and own tests, but I unchained them.
3. Done